config/optional/metatag.metatag_defaults.node.yml

```diff
@@ -8,3 +8,4 @@ tags:
   description: '[node:summary]'
   canonical_url: '[node:url]'
   og_image: '[node:field_featured_image:entity:field_media_image:entity:url]'
+  og_locale: 'en-au'
```

Comment:
I actually don't think we use en-au anywhere.

This page here - https://digital-vic.atlassian.net/servicedesk/customer/article/2272657746 - lists the documentation that the content team use for managing this field and the codes they use. A good one might be 'en' for an example - but we don't need to set a default at all here.

The default for our english content should be an empty field.

Reply:
Hi @krakerag and @MdNadimHossain , I have updated my pr for your reviews. Ta